src/routers/router.js

```diff
@@ -11,6 +11,7 @@ import validateAdminApiKey from '../middleware/validateAdminApiKey';
 import attachRedisClient from '../middleware/attachRedisClient';
 import analytics from '../middleware/analytics';
 import authorization from '../middleware/authorization';
+import { login } from '../controllers/auth/login';
```

Comment:
nit: can you move this import to be right below `getStats` so all the controller imports are near each other?